web/styles/message_row.css

```diff
@@ -782,6 +782,12 @@
     }
 
     .message-edit-textbox {
+        /* Setting `position:relative` on this container ensures that
+        the copy button (its immediate child with `position: absolute`)
+        is positioned relative to this container instead of <html>.
+        See https://chat.zulip.org/#narrow/channel/137-feedback/topic/Copy.20button.20for.20View.20Original.20Message/near/2143001
+        for a better explanation. */
+        position: relative;
```

Comment:
Generally it's best to write text that is a clear explanation without referencing a CZO thread.